includes/admin/class-wc-admin-duplicate-product.php

```diff
@@ -43,6 +43,18 @@ public function dupe_link( $actions, $post ) {
 			return $actions;
 		}
 
+		// Add Class to Delete Permanently link in row actions.
+		$total_sales = get_post_meta( $post->ID, 'total_sales', true );
```

Comment:
This should be using CRUD, `get_total_sales` from a product object.

Reply:
Implemented Change